contentcuration/contentcuration/db/models/sql/query.py

```diff
@@ -0,0 +1,63 @@
+from django.db import connections
+from django.db.models.sql import Query
+
+from contentcuration.db.models.sql.compiler import SQLInsertFromCompiler
+from contentcuration.db.models.sql.compiler import SQLUpdateFromCompiler
+
+
+class InsertFromQuery(Query):
```

Comment:
Could you add some comments on how to use this and the UpdateFromQuery?

Reply:
Good point, I didn't add any comments regarding these. Will do.